core/templates/dev/head/App.ts

```diff
@@ -306,13 +306,13 @@ if (typeof Object.create !== 'function') {
     var F = function() {};
     Object.create = function(o) {
       if (arguments.length > 1) {
-        throw Error('Second argument for Object.create() is not supported');
+        throw new Error('Second argument for Object.create() is not supported');
```

Comment:
I think this change is unnecessary. This fix should only be limited to places that throw primitive data types. Not all occurences of throw. Please undo this change and ditto below.